meson.build

```diff
@@ -167,6 +167,7 @@ foreach a,v: test_arr
   e = executable('test_' + a,
     sources: v,
     include_directories: [lib_inc],
+    link_with: libiscsi_usr,
     dependencies: libiscsi_usr_dep,
```

Comment:
libiscsi_usr_dep is a container object that specifies `link_with: libiscsi_usr`, so this is redundant and the executables already link to libiscsi_usr.